js/admin.js

```diff
@@ -0,0 +1,31 @@
+jQuery( document ).ready(function($) {
+	$('#check-status').click(function(){
+		$(".theme-check-report").empty();
+		themeCheckRunPHPCS();
+	});
+	function themeCheckRunPHPCS(){
+		$.ajax({
+			url: themeCheck.ajax_url,
```

Comment:
It did not work for me. I expect the PHP function needs to return the progress. 
There is a way mentioned https://secure.php.net/manual/en/function.exec.php#88704 but I have not understood it fully.